numpy/distutils/fcompiler/gnu.py

```diff
@@ -182,6 +199,11 @@ def get_library_dirs(self):
                         if os.path.exists(path):
                             opt.append(d2)
                 opt.append(d)
+        # For Macports / Linux, libgfortran and libgcc are not co-located
+        if sys.platform[:5] == 'linux' or sys.platform == 'darwin':
```

Comment:
No need for this `if`, as it's handled within the function already.

Reply:
Indeed - I suppose I was trying to mirror the structure of the existing `get_library_dirs()` methods, but having the `if` statement here is probably a bad idea. To add other platforms somebody would need to change code in 3 places as opposed to just one. Since this is already merged, should I take if `if`s out and submit another PR?